src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Providers/AbstractSemanticQuickInfoProvider.cs

```diff
@@ -266,16 +273,28 @@ internal abstract partial class AbstractSemanticQuickInfoProvider : AbstractQuic
             return CreateDocumentationCommentDeferredContent(null);
         }
 
+        protected abstract bool GetBindableNodeForTokenIndicatingLambda(SyntaxToken token, out SyntaxNode found);
```

Comment:
To check: this isn't being used by TypeScript or F# that this is a breaking change to derivers? I don't see _how_ they could , but want to confirm.

Reply:
I checked with F# folks (Will and Brett) and they do not reference `AbstractSemanticQuickInfoProvider` from Roslyn (they only depend on some interfaces for quick info provider.

Same thing with TypeScript (Mine Starks), which only depends on interfaces, but not this abstract type.